float32 curSpeed
float32 distObstacle
float32 aExpect
float32 speedExpect
